uk_bin_collection/uk_bin_collection/councils/WiltshireCouncil.py

```
@@ -121,18 +121,23 @@ def parse_data(self, page: str, **kwargs) -> dict:
+                    # A day can have more than one event container (e.g. a
+                    # garden waste subscriber whose garden collection falls
+                    # on the same day as recycling) - collect all of them,
+                    # not just the first.
+                    for event_container in event.select(".rc-event-container"):
+                        span = event_container.select_one("span")
+                        if not span:
+                            continue
+
+                        collection_types = span.text.strip().split(" and ")
+
+                        for type in collection_types:
```

Ruff (0.15.20)

[error] Variable `type` is shadowing a Python builtin

(A001)